lib/tasks/router.rake

```diff
@@ -13,33 +13,6 @@ namespace :router do
     @router_api.add_backend(@app_id, url)
   end
 
-  task :register_routes => :router_environment do
-    [
-      %w(/favicon.ico exact),
-      %w(/humans.txt exact),
-      %w(/robots.txt exact),
-      %w(/google6db9c061ce178960.html exact), # DWP YouTube annotations
-      %w(/google991dec8b62e37cfb.html exact),
-      %w(/googlef35857dca8b812e7.html exact),
-      %w(/apple-touch-icon.png exact static),
-      %w(/apple-touch-icon-144x144.png exact),
-      %w(/apple-touch-icon-114x114.png exact),
-      %w(/apple-touch-icon-72x72.png exact),
-      %w(/apple-touch-icon-57x57.png exact),
-      %w(/apple-touch-icon-precomposed.png exact),
-    ].each do |path, type|
-      begin
-        puts "Registering #{type} route #{path}"
-        @router_api.add_route(path, type, @app_id)
-      rescue => e
-        puts "Error registering route: #{e.message}"
-        raise
-      end
-    end
-
-    @router_api.commit_routes
-  end
-
   desc "Register application and routes with the router"
-  task :register => [ :register_backend, :register_routes ]
+  task :register => :register_backend
```

Comment:
`register_backend` shouldn't be needed any more now that this isn't registering any routes.  It shouldn't be talking directly to the router at all.